tests/integration/controller/withwatcher/watcher_controller_helper_test.go

```diff
@@ -283,3 +284,32 @@ func listenerHTTPRouteExists(ctx context.Context, clt *istio.Client, namespace s
 
 	return errRouteNotFound
 }
+
+func isRouteConfigEqual(route1 *istioapiv1beta1.HTTPRoute, route2 *istioapiv1beta1.HTTPRoute) bool {
```

Comment:
Used to be part of client, but was only used in this test file. So I opted for moving it into the test.